integrations/aws/utils/aws.py

```diff
@@ -4,35 +4,22 @@
 from port_ocean.context.ocean import ocean
 from starlette.requests import Request
 
-from aws.session_manager import SessionManager, ASSUME_ROLE_DURATION_SECONDS
+from aws.session_manager import SessionManager
+from aws.auth_provider import (
+    OrganizationCredentialsProvider,
+)
 from aws.aws_credentials import AwsCredentials
 
-from aiocache import cached, Cache  # type: ignore
-from asyncio import Lock
-
 from port_ocean.utils.async_iterators import stream_async_iterators_tasks
 
-_session_manager: SessionManager = SessionManager()
-
-CACHE_DURATION_SECONDS = (
-    0.80 * ASSUME_ROLE_DURATION_SECONDS
-)  # Refresh role credentials after exhausting 80% of the session duration
 
-lock = Lock()
+provider = OrganizationCredentialsProvider()
+_session_manager: SessionManager = SessionManager(provider)
```

Comment:
So we always use the OrganizationCredentialsProvider even on a single account mode?